relay/sources/relayd/src/remote_run.rs

```diff
@@ -65,32 +65,66 @@ impl RemoteRun {
         })
     }
 
+    pub fn target_to_vector(&self, job_config: Arc<JobConfig>) -> Vec<String> {
```

Comment:
This should be a method of RemoteRunTarget, called `get_connected_nodes` or something like this.

Reply:
Vector is a technical implementation detail, you should strive to use applicative business names as much as possible to keep things readable